### DISCUSSION TITLE
My account has been suspended, what happend?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#39122] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi,

    My account @xxrz has been suspended, what happend, coule you tell me how to solve it?
    The account is very important to me, and I sincerely look forward to your replay!

Thank you.

**Comment by radar-blip** [comment, marked as answer]
For account related issues, see here: [Support page](https://support.github.com/contact/account)

**Comment by SanderKofficial** [comment]
This can happen because you did something, or you have a large ping rate or other things that caused the system to suspend you.

But ask [support](https://support.github.com/contact/account)for help.

**Reply by ghost** [reply]
Why isn't github telling users what the reason for the suspension is?

Shame on github for being a platform for false and unfounded accusations and punishments. 
Hardly Christian. Far from human and humane.
